pkg/apis/core/validation/validation_test.go

```diff
@@ -815,6 +830,7 @@ func TestValidatePersistentVolumeClaim(t *testing.T) {
 					},
 				},
 				StorageClassName: &validClassName,
+				VolumeMode:       &validMode,
```

Comment:
why was this change required?

Reply:
this "good-claim" test-case seems to just have every valid field dumped into it so i added validMode. unrelated to the other test changes

Reply:
ok